hypershift-operator/controllers/hostedcluster/hostedcluster_webhook.go

```diff
@@ -75,28 +130,33 @@ func validateNetworkCIDRs(hc *hyperv1.HostedCluster) field.ErrorList {
 	serviceCIDR := hc.Spec.Networking.ServiceCIDR
 	machineCIDR := hc.Spec.Networking.MachineCIDR
 
-	// Validate CIDR format..
+	// I think if these are all unset we should ignore them.
```

Comment:
We should use the new slice fields in hc.Spec.Networking (https://github.com/openshift/hypershift/blob/main/api/v1alpha1/hostedcluster_types.go#L487-L507)

Reply:
This is the behavior with this in it.  We use a separate function validateSliceNetworkCIDRs() to validate those.  I'll change the comment to be more appropriate.  I've already seen a few failures from this not being set.